pkg/templates/charts/toggle/multicloud-operators-subscription/templates/argocd-pull-integration-controller-manager.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: argocd-pull-integration
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/part-of: argocd-pull-integration
+    control-plane: controller-manager
+  name: argocd-pull-integration-controller-manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      creationTimestamp: null
+      labels:
+        control-plane: controller-manager
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Why is this required-during-scheduling node affinity spec necessary here?  I have start to see something similar in some other non-ACM things, but we do not customarily include such a thing in ACM or MCE deployments....wondering where it comes from, what the use case is that requires it, and if its somethign we should be adding across the board for ACM and MCE if the use case is relevant to us.

Reply:
Thanks for the review Joe. We will remove this additional deployment.
This came from the kubebuilder project bootstrap.